Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.11-slim
+LABEL maintainer="jacktyler2391@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
```

Comment:
There is a typo in the environment variable name. It should be `PYTHONUNBUFFERED` instead of `PYTHOUNNBUFFERED`.